src/GraphQLParser.Tests/Validation/LexerValidationTests.cs

```diff
@@ -391,5 +405,33 @@ public void Lex_UnterminatedStringWithText_ThrowsExceptionWithCorrectMessage()
             exception.Line.ShouldBe(1);
             exception.Column.ShouldBe(14);
         }
+
+        [Fact]
+        public void Lex_UnterminatedBlockString_ThrowsExceptionWithCorrectMessage()
+        {
+            var exception = Should.Throw<GraphQLSyntaxErrorException>(() => "\"\"\"".Lex());
+
+            exception.Message.ShouldBe(
+                "Syntax Error GraphQL (1:4) Unterminated string.\n" +
```

Comment:
@Shane32  I propose to change error message to `Unterminated block string`. Also you used `BlockString` term in other message. I propose to change it to `block string` since spec does not define term `BlockString`. It defines `BlockStringCharacter` and `BlockStringValue` function.

Reply:
![изображение](https://user-images.githubusercontent.com/21261007/140186141-5534b6aa-7e93-4c90-88e9-7f45643cf6cf.png)
the only place where it is mentioned

Reply:
I don't think anyone cares, but sure, go ahead and make a PR if you like.  (I personally think it's fine as-is.)

Reply:
https://github.com/graphql/graphql-spec/issues/899